OpenEdXMobile/src/main/java/org/edx/mobile/whatsnew/WhatsNewItemModel.java

```diff
@@ -3,12 +3,38 @@
 import android.os.Parcel;
 import android.os.Parcelable;
 import android.support.annotation.NonNull;
+import android.text.TextUtils;
 
 import java.util.List;
 
 public class WhatsNewItemModel implements Parcelable {
-    public static final String PLATFORM_ANDROID = "android";
-    public static final String PLATFORM_IOS = "ios";
+    public enum Platform {
```

Comment:
`Platform` can be renamed to `SupportedPlatforms`.

Reply:
In java we use singular name for enum names so can't put `s` at end. Other than that `Platform` still looks good to me as its more generic and already composed within `WhatsNewItemModel` class.